Realm/RLMUserAPIKey.mm

```diff
@@ -51,7 +51,7 @@ - (NSString *)name {
 // the response when an API key is first created.
```

Comment:
I'm a little curious about this text over here, this will only return only when we call the `create_api_key` object store API, and will return nil when we fetch `fetch_api_key` and `fetch_api_keys`

Reply:
This is intentional, as the API key is not supposed to be revealed outside of the `create` action, as it's a secret. See docs here: https://www.mongodb.com/docs/realm/sdk/swift/app-services/manage-user-api-keys/

Reply:
Thanks @matthewmorek for the info.